README.md

```diff
@@ -105,36 +105,27 @@ DOCKER IMAGES
 Docker images are build by travis-ci and pushed to [hub.docker.com](https://hub.docker.com/r/ergw/ergw-c-node/tags),
 and by gitlab.com and pushed to [quay.io](https://quay.io/repository/travelping/ergw-c-node?tab=tags).
 
-BUILDING
---------
-
-*The minimum supported Erlang version is 22.3.4.*
-
-Erlang 23.0.3 is the recommended version.
-
-Using rebar:
-
-    # rebar3 compile
-
-RUNNING
+BUILDING & RUNNING
 -------
-
+Required:
+* The minimum supported Erlang version is **22.3.4** (*Erlang 23.0.3 is the recommended version.*).
+* [Rebar3](https://www.rebar3.org/)
 An *erGW* installation needs a user plane provider to handle the GTP-U path. This
 instance can be installed on the same or different host.
 
 A suitable user plane node based on [VPP](https://wiki.fd.io/view/VPP) can be found at [VPP-UFP](https://github.com/travelping/vpp/).
 
-*erGW* can be started with the normal Erlang command line tools, e.g.:
-
-```
-erl -setcookie secret -sname ergw -config ergw.config
-Erlang/OTP 19 [erts-11.0.1] [source] [64-bit] [async-threads:10] [kernel-poll:false]
+**ERGW** can be started with **rebar3** command line tools, and build with run can looks like:
```

Comment:
capitalization of the name is usually `erGW` for erlang-GateWay

Reply:
Done.